sdk/python/lib/pulumi/automation/_local_workspace.py

```diff
@@ -216,6 +216,16 @@ def post_command_callback(self, stack_name: str) -> None:
         # Not used by LocalWorkspace
         return
 
+    def add_environments(self, stack_name: str, *environment_names: str) -> None:
+        args = ["config", "env", "add"]
```

Comment:
Similar to my feedback https://github.com/pulumi/pulumi/pull/14788/files#r1419956650

Should we check the version and provide a nicer error if it's an older CLI that doesn't support the commands?

https://github.com/pulumi/pulumi/blob/e071d9da9265a2db4a871c7bcb6d93e508f5964d/sdk/python/lib/pulumi/automation/_local_workspace.py#L317-L330